modules/opencv/4.11.0/overlay/BUILD.bazel

```
@@ -0,0 +1,361 @@
+load("//:opencv.bzl", "CONFIG", "opencv_module")
+load("@bazel_skylib//rules:write_file.bzl", "write_file")
+load("@bazel_skylib//rules:expand_template.bzl", "expand_template")
+
+write_file(
+    name = "_private/version_string.inc",
+    out = "private/version_string.inc",
+    content = ["\"\""],
```

Nit: Should this be the actual version?
```suggestion
    content = ['"4.11.0"'],
```